api/src/main/resources/workbench-api.yaml

```diff
@@ -3284,7 +3284,7 @@ paths:
           description: A collection of criteria menu options
           schema:
             "$ref": "#/definitions/CriteriaMenuListResponse"
-  "/v1/cohortbuilder/{workspaceNamespace}/{workspaceId}/criteria/domaininfo":
+  "/v1/cohortbuilder/{workspaceNamespace}/{workspaceId}/criteria/domaininfoDepreciate":
```

Comment:
Feels nicer to leave this as domaininfo and to change new endpoint to be domaincard. This is more consistent with db table naming as well.

Reply:
Done